file-formats/nont/type/zif_aff_nont_v1.intf.abap

```diff
@@ -0,0 +1,35 @@
+INTERFACE zif_aff_nont_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">General Information</p>
+    BEGIN OF ty_main,
+      "! $required
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
+
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header         TYPE zif_aff_types_v1=>ty_header_60_cloud,
+
+      "! <p class="shorttext">Name</p>
+      "! Node type name in camel case
+      "! $required
+      name           TYPE c LENGTH 30,
+
+      "! <p class="shorttext">Expanded Name</p>
+      "! Expanded name
+      "! $required
+      expanded_name  TYPE c LENGTH 512,
+
+      "! <p class="shorttext">SAP Object Type</p>
+      "! Name of SAP Object Type
+      parent_name    TYPE c LENGTH 30,
```

Comment:
Typically, if you choose a different title/description than the field name you want to adapt one either the title or the field name ;)

Reply:
You are absolutely right about this. I will change the field "parent_name" to "sap_object_type".